examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py

```diff
@@ -0,0 +1,68 @@
+"""
+Environment Setting
+
+Enable Intel Optimized TensorFlow 2.6.0 and newer by setting environment variable TF_ENABLE_ONEDNN_OPTS=1
+That will accelerate training and inference, and  it's mandatory requirement of running Intel® Neural Compressor quantize Fp32 model or deploying the quantized model.
+"""
+
+import neural_compressor as inc
+print("neural_compressor version {}".format(inc.__version__))
+
+import tensorflow as tf
+print("tensorflow {}".format(tf.__version__))
+
+from neural_compressor.experimental import Quantization, common
+
+import numpy as np
+import tensorflow_datasets as tfds
+
+
+
+# define class number
+class_num=3
+
+# define input image size and class number
+w=h=32
+
+
+def preprocess(image, label):
+    image = tf.cast(image, tf.float32)/255.0
+    return tf.image.resize(image, [w, h]), tf.one_hot(label, class_num)
+
+
+def load_raw_dataset():
+    raw_datasets, raw_info = tfds.load(name = 'beans', with_info = True,
+                                       as_supervised = True, 
+                                       split = ['train', 'test'])
+    return raw_datasets, raw_info
+    
+class Dataset(object):
+    def __init__(self):
+        datasets , info = load_raw_dataset()        
+        self.test_dataset = [preprocess(v, l) for v,l in datasets[-1]]
+    
+    def __getitem__(self, index):
+        return self.test_dataset[index]
+
+    def __len__(self):
+        return len(list(self.test_dataset))
+
+def auto_tune(input_graph_path, yaml_config, batch_size, int8_pb_file):
+    quantizer = Quantization(yaml_config)
```

Comment:
we can get this example merged at first, and then submit new PR to replace those deprecated API usage to 2.0 new API

Reply:
yes.
When 2.0 is published, I will updated all example with new API.
That need different PR for different example.